CHANGELOG.md

```diff
@@ -15,8 +39,7 @@
    ([#8441](https://github.com/Microsoft/vscode-jupyter/issues/8441))
 1. Add support for pre-release version of Extensions for VS Code Insiders.
    ([#8463](https://github.com/Microsoft/vscode-jupyter/issues/8463))
-1. Allow to retrieve the jupyter filename programmatically (__vsc_ipynb_file__) [fix support for ClearML]
-   ([#8475](https://github.com/Microsoft/vscode-jupyter/issues/8475))
+1. Allow to retrieve the jupyter filename programmatically (**vsc_ipynb_file**) [fix support for ClearML](<[#8475](https://github.com/Microsoft/vscode-jupyter/issues/8475)>)
```

Comment:
Why the angle brackets?

Reply:
I didn't add any of these changes to the changelog. Some markdown auto editing thing did.